test/unit/controllers/ImportContactModalControllerSpec.js

```diff
@@ -0,0 +1,190 @@
+'use strict'
+/* global describe, it, inject, expect, beforeEach, jasmine */
+
+describe('ImportContactModalController', function () {
+  beforeEach(module('myApp.controllers'))
+
+  beforeEach(inject(function (_$controller_, _$rootScope_) {
+    this.$controller = _$controller_
+    this.$rootScope = _$rootScope_
+
+    this.pcs = {
```

Comment:
`pcs` stands for [Post Concert Syndrom](http://www.urbandictionary.com/define.php?term=PCS), doesn't it?